pkg/controller/utils/utils.go

```diff
@@ -161,3 +147,47 @@ const (
 func GetConcurrentReconciles() int {
 	return concurrentControllerReconciles
 }
+
+// MergeJsons will merge the global and local pull secret and return it
+func MergeJsons(globalPullSecret string, localPullSecret string, cdLog log.FieldLogger) (string, error) {
+
+	type dockerConfig map[string]interface{}
+	type dockerConfigJSON struct {
+		Auths       dockerConfig      `json:"auths"`
+		HTTPHeaders map[string]string `json:"HttpHeaders,omitempty"`
```

Comment:
I am still of a strong belief that this should be removed unless we are going to merge it.